subxt/examples/blocks_subscribing.rs

```diff
@@ -24,8 +23,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         println!("  Extrinsics:");
 
         // Log each of the extrinsic with it's associated events:
-        let body = block.body().await?;
-        for ext in body.extrinsics().iter() {
+        let extrinsics = block.extrinsics().await?;
```

Comment:
Nice simplification for the end user!

If (ever) justification turns to be a thing in the future we could expose it on the same level as `extrinsics` (ie block.extrinsics())

Reply:
Yup definitely; if they are a thing then they won't be in the block body I guess anyway (since the body of a block afaik is just the extrinsics). I'm not actually sure where they live offhand; maybe theres a storage or runtime API call to fetch them (I haven't looked!)